doc/lsst.pipe.base/command-line-task-dataid-howto.rst

```diff
@@ -0,0 +1,203 @@
+.. _command-line-task-dataid-howto:
+
+###########################################
+Specifying data IDs with command-line tasks
+###########################################
+
+:ref:`Command-line tasks <using-command-line-tasks>` consume data and output new data products.
+You can tell a command-line task exactly what data to process by passing data IDs to the task's :option:`--id` argument.
+The task combines the data IDs you provide with the dataset types it expects to retrieve specific datasets for processing with the Butler.
+
+This page describes how to use the :option:`--id` argument to select data for command-line task processing.
+This documentation also applies to the :option:`--selectId` argument for many-to-one processing tasks.
+
+.. _command-line-task-dataid-howto-about-dataid-keys:
+
+About data ID keys
+==================
+
+Individual data IDs are composed of a set of keys and their values.
+The set of keys that are included in a data ID depend on the dataset type and even the observatory package.
+The important categories of data IDs are for exposures (generally, individual observations from an observatory) and coadds.
```

Comment:
Exposures and Coadds mean very specific things in LSST-speak, and it would be good to add links to more comprehensive definitions here.